lib/asciidoctor-revealjs/highlightjs.rb

```diff
@@ -33,8 +35,317 @@ def docinfo location, doc, opts
           else
             theme_href = "#{revealjsdir}/lib/css/zenburn.css"
           end
-          %(<link rel="stylesheet" href="#{theme_href}"#{opts[:self_closing_tag_slash]}>)
+          base_url = doc.attr 'highlightjsdir', %(#{opts[:cdn_base_url]}/highlight.js/#{HIGHLIGHT_JS_VERSION})
```

Comment:
Maybe we should document this attribute? It allows to use a local install of highlight.js (rather than using the CDN). 
It's useful when you want to use highlight.js syntax highlighter in offline mode.

Reply:
Yes, definitely!

Reply:
done